### PR TITLE
chore: release 2.7.0

### LINKED ISSUE #218: support django 3.2
Please update the package to support Django version 3.2 as well

### PR DESCRIPTION
**Pull request by Radi85**
closes #218

### CODE REVIEW

File: docs/source/Changelog.rst [hunk 1]
```diff
@@ -1,6 +1,17 @@
 Changelog
 =========
 
+2.7.0
+------
+
+- Add support for django `3.2`.
```
Comment (abhiabhi94) [inline docs/source/Changelog.rst:7]:
```suggestion
- Add support for django ``3.2``.
```

File: docs/source/Changelog.rst [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+- Allow blocking users/emails from adding/reacting with comments.
+- Improve commenting anonymously UI.
+- Enhance API docs by adding openapi and swagger page to RTD.
```
Comment (abhiabhi94) [inline docs/source/Changelog.rst:10]: i am not a big fan of using acronyms without defining it at some place. Not everyone may get the context at all times.
Maybe we can add the link to the `readthedocs` page here.

File: docs/source/usage.rst [hunk 3]
```diff
@@ -213,7 +213,7 @@ PS: This feature needs the email settings to be configured similar to `4. Allow
 7. Enable blocking system
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Blocking functionality is added in version 2.7.0. It allows moderators to block users/emails from creating/editing or reacting to a comment.
+Blocking functionality is added in version `2.7.0`. It allows moderators to block users/emails from creating/editing or reacting to a comment.
```
Comment (abhiabhi94) [inline docs/source/usage.rst:216]: This change is not necessarily related to the release. I would make this in a separate commit with something like `Correct Typo in ...`.

**Review by abhiabhi94 — COMMENTED** [review]
Some small nit-pickings. Looks good to me.

**Comment by abhiabhi94** [comment]
Apart from all this code, amidst the pandemic, shifting and everything else, how are you @Radi85, and how is everything at home?

File: docs/source/usage.rst (hunk 3 above)
Comment (abhiabhi94) [reply docs/source/usage.rst:216]: you may still make the commit in this pull-request itself.

**Comment by Radi85** [comment]
Thanks for your check!

**Comment by Radi85** [comment]
> Apart from all this code, amidst the pandemic, shifting and everything else, how are you @Radi85, and how is everything at home?

I am finally settling down in the new apartment and getting back to my old routine slowly 😄 
How about you? How's life going there?

File: docs/source/Changelog.rst (hunk 2 above)
Comment (abhiabhi94) [reply docs/source/Changelog.rst]: i was originally talking about the RTD acronym used here. But, may this clean-up can be done in a separate one, as this will block the release.

**Review by abhiabhi94 — APPROVED** [review]
Thanks for the updates.

**Comment by abhiabhi94** [comment]
> How about you? How's life going there?

I have recently brought a dog home. She has messed up my routine a lot. But, the other parts with her seem enjoyable at the moment. She keeps me distracted due to her demands for attention. 😄

**Comment by abhiabhi94** [comment]
I think this need to be merged only after #226.

**Comment by abhiabhi94** [comment]
travis is taking forever to start this! 💢

**Comment by abhiabhi94** [comment]
I see a lot of `403` status on the network tab of developer console for the request `GEThttps://api.travis-ci.com/v3/build_backups?repository_id=14692409&offset=0&limit=1` on the page https://app.travis-ci.com/github/Radi85/Comment/jobs/532506604. Maybe the quota for the builds has been exhausted on travis or it has something to do with the integration between travis and github?

**Comment by Radi85** [comment]
>Maybe the quota for the builds has been exhausted on travis or it has something to do with the integration between travis and github?

I think quota issue. Can you please review/approve the PR so that I can proceed with the release once the pipeline is done?

**Review by abhiabhi94 — APPROVED** [review]
Thanks @Radi85, for the updates back and forth.

**Comment by abhiabhi94** [comment]
hey @Radi85, should we consider switching to github actions? the only point of contention being the release. if you want the release process to still be automated, you probably will have to add your PyPI password as github secret.  i have some trust issues with github since it has happened several times that secrets have been leaked. i would suggest switching to manual releases for the time being, and i can take that responsibility for now, if there's an issue with it. in case you're okay with using github secrets, then you can add the secret to the repository. i will make a pull-request soon, and we can continue this discussion there.

**Review by abhiabhi94 — APPROVED** [review]
Yay, 🚀  finally, `2.7.0` is going to be released 🥳 . Thanks @Radi85 for your efforts.
